kernel/kernel-api/src/main/java/io/delta/kernel/utils/FileStatus.java

```diff
@@ -68,6 +76,24 @@ public long getModificationTime() {
         return modificationTime;
     }
 
+    /**
+     * Get the tags of the file.
+     *
+     * @return the file tags
+     */
+    public Map<String, String> getTags() {
+        return tags;
+    }
+
+    /**
+     * Get the stats of the file.
+     *
+     * @return the file stats
+     */
+    public String getStats() {
+        return stats;
+    }
+
```

Comment:
`FileStatus` is a public API and we can't have these fields in `FileStatus`. If you need an object with file info, stats, and tags, you can create one in your connector and write a utility method similar to the `InternalScanFileUtils.getAddFileStatus`.